### PR TITLE
Add docs for live HTML preview of documentation and clean up.

### LINKED ISSUE #673: Add `sphinx-autobuild`
When I edit docs, I like when the reload to show how my changes render. I use [`sphinx-autobuild`](https://pypi.org/project/sphinx-autobuild/) for all my documentation.

### PR DESCRIPTION
**Pull request by stevepiercy**
- Use shell instead of bash as a universal shell.
- Use pycon for Python console commands.
- Remove command prompts from shell commands to make them easier to copy and have neutral shell.
- Remove `http.server` option as it's been replaced by `make livehtml`.

Closes #673

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-  pip install icalendar
+.. code-block:: shell
+
+    pip install icalendar
 
 If installation is successful, you will be able to import the iCalendar
-package, like this::
+package, like this:
+
+.. code-block:: pycon
```
Comment (niccokunzmann) [inline docs/install.rst:13]: nice, I did not know pycon existed.

File: docs/install.rst [hunk 2]
```diff
@@ -106,40 +110,37 @@ manually setup ``icalendar`` like this.
...
-    $ source .tox/py39/bin/activate
-    $ pip install -r requirements_docs.txt
-    $ cd docs
-    $ make html
+You can now open the output from ``_build/html/index.html``.
 
-You can now open the output from ``_build/html/index.html``. To build the
-presentation-version use ``make presentation`` instead of ``make html``. You
-can open the presentation at ``presentation/index.html``.
+To build the documentation, view it in a web browser, and automatically reload changes while you edit documentation, use ``make livehtml`` instead of ``make html``.
```
Comment (niccokunzmann) [inline docs/install.rst]: personally, for the easy copy&paste style, I would prefer those commands to do into a codeblock.

Comment (niccokunzmann) [reply docs/install.rst]:
```
.. code-block:: shell

    make livehtml 
```

Then, it is consistent in style and suggests that it is of equal value.

Comment (niccokunzmann) [reply docs/install.rst]: Could you do that? I think it is fine to merge already.

**Review by niccokunzmann — APPROVED** [review]

File: docs/install.rst (hunk 2 above)
Comment (stevepiercy) [reply docs/install.rst]: I could do one even better. Would you find this useful?

https://sphinx-copybutton.readthedocs.io/en/latest/index.html

I use it in Plone docs.

Comment (stevepiercy) [reply docs/install.rst]: I'll reformat as code block, and put the plug-in in a new issue and PR.

Comment (stevepiercy) [reply docs/install.rst]: Done in 8581d355d6250b27ca9d8708d7c206f66cee73e4

**Pull request merged**